### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Richa-rani02**
Added Auth pages and modal
1. Login and signup page
2. rules modal

### CODE REVIEW

**Review by rehanmohiuddin — APPROVED** [review]

File: src/pages/index.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export {Home} from "./HomePage/Home";
```
Comment (deepak29-git) [inline src/pages/index.js:1]: this is good approach that you have export all files from index.js file

File: src/pages/Authentication/Auth.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+.auth-form{
+    width: 25rem;
+    display: flex;
+    padding: var(--spacing-2);
+    background-color: var(--green-1);
+    align-items: center;
+
+}
+.auth-form h4 {
```
Comment (deepak29-git) [inline src/pages/Authentication/Auth.css:21]: keep consistent naming convention either using BEM or OOCSS ,choose one of them

File: src/components/Modal/Modal.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    setModalActive(prev=>!prev);
+    navigate("/question");
+ }
+    return (
+        <div class={`modal-container ${isModalActive && 'modal-active'}`}>
+            <div class="popup">
+                <div class="modal-content display-flex-col">
+                    <h2 class="heading-1 bottom-gutter-md center-text">RULES 📑</h2>
+                    <div className="rules-list">
+                        {rules.map((rule) => (
```
Comment (deepak29-git) [inline src/components/Modal/Modal.jsx:16]: suggestion : you can also extract object properties

**Review by deepak29-git — APPROVED** [review]
you folder structure is also good.

**Pull request merged**